ReactWindows/js/Components/WindowsProgressRing/ProgressRingWindows.ios.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ProgressRingWindows
+ */
+'use strict';
+
+module.exports = require('UnimplementedView');
```

Comment:
Use empty view here as well to match ProgressBarAndroid.ios.js and ActivityIndicatorIOS.android.js.